test/export/test_export.py

```diff
@@ -4160,6 +4169,11 @@ def false_fn(x):
         dim0 = torch.export.Dim("dim0", min=3)
         inp = torch.ones(6, 4)
         ep = export(Foo(), (inp,), dynamic_shapes={"x": {0: dim0}})
+        schema = get_hop_schema(ep)
+        self.assertExpectedInline(
+            str(schema),
+            """cond(SymBool pred, GraphModule true_fn, GraphModule false_fn, Tensor[2] operands) -> Tensor[1] output""",
```

Comment:
1) does export support Tensor[2] (instead of Tensor[]). If it does not, do we want to make export support this? (or just change it back to Tensor[])

Reply:
Discussed with a few folks. The length can be used for better runtime input checking in Sigmoid. The conclusion is that we could keep it there and strip away the size if it's not required.